specs/commandline/plugin.md

```diff
@@ -99,6 +209,6 @@ An example of output from `notation plugin list`:
 
 ```text
 NAME                                   DESCRIPTION                                   VERSION       CAPABILITIES                                                                                            ERROR
-azure-kv                               Sign artifacts with keys in Azure Key Vault   v0.5.0-rc.1   [SIGNATURE_GENERATOR.RAW]                                                                                <nil>
-com.amazonaws.signer.notation.plugin   AWS Signer plugin for Notation                1.0.290       [SIGNATURE_GENERATOR.ENVELOPE SIGNATURE_VERIFIER.TRUSTED_IDENTITY SIGNATURE_VERIFIER.REVOCATION_CHECK]   <nil>
+azure-kv                               Sign artifacts with keys in Azure Key Vault   v1.0.0        Signature generation                                                                                <nil>
+com.amazonaws.signer.notation.plugin   AWS Signer plugin for Notation                1.0.290       Signature envelope generation, Trusted Identity validation, Certificate chain revocation check   <nil>
```

Comment:
Should we keep capabilities as enum instead of text as they are more for plugin publisher as compared to consumer?

Reply:
The original enum could be used and kept on the [plugin-extensibility.md#plugin-metadata)](https://github.com/notaryproject/specifications/blob/main/specs/plugin-extensibility.md#plugin-metadata). For the output of `notation plugin list`, I think it should be user-friendly so using text would be much straightforward.